docs/development/getting_started.rst

```diff
@@ -54,24 +54,34 @@ Either way, once you've installed Vagrant you should run:
 
 Finally, install Ansible so it can be used with Vagrant to automatically
 provision VMs. We recommend installing Ansible from PyPi with ``pip`` to ensure
-you have the latest stable version.
+you have the latest stable version:
 
 .. code:: sh
 
     sudo apt-get install python-pip
-    sudo pip install -U ansible
 
-If you're using Ubuntu, you can install a sufficiently recent version of
-Ansible from backports (if you prefer): ``sudo apt-get install
-ansible/trusty-backports``
+Since the version of ansible required to provision SecureDrop VMs may flux out
+of sync with the version in your distro's repos, we recommend installing the
+latest stable ansible to a virtual environment using `virtualenvwrapper` or
```

Comment:
That's correct, we definitely _don't_ want to recommend 2.x—at least not yet. Let's simply leave the "stable" comment out and state the version, i.e. 

>  we recommend installing Ansible v1.8.4 in a virtualenv. 

Doesn't matter whether folks use `virtualenv` or `virtualenvwrapper`.